tests/junit.test.ts

```diff
@@ -0,0 +1,46 @@
+import {TestRun, Status} from '../src'
+
+describe('toJUnitObj', function () {
+    const run = new TestRun();
+
+    beforeEach(function() {
+        const s1 = run.withSuite('somegroup')
+        const s2 = s1.withSuite('somefile.test.js')
+        s2.withTest('yay', {
```

Comment:
Would be great to include metadata field as well 🙏

Reply:
added